### PR TITLE
Add Hirak FaceAPI to Machine Learning

### PR DESCRIPTION
**Pull request by FayeNichols**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1168,6 +1168,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Deepcode](https://www.deepcode.ai) | AI for code review | No | Yes | Unknown |
 | [Dialogflow](https://cloud.google.com/dialogflow/docs/) | Natural Language Processing | `apiKey` | Yes | Unknown |
 | [EXUDE-API](http://uttesh.com/exude-api/) | Used for the primary ways for filtering the stopping, stemming words from the text data | No | Yes | Yes |
+| [FaceAPI](https://faceapi.hirak.site/) | Face detection, face recognition with age estimation/gender estimation, accurate, no quota limits | `apiKey` | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:1171]: Rename API name to Hirak FaceAPI

```suggestion
| [Hirak FaceAPI](https://faceapi.hirak.site/) | Face detection, face recognition with age estimation/gender estimation, accurate, no quota limits | `apiKey` | Yes | Unknown |    
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your new contribution @FayeNichols 😄 

Can you resolve this review, please?

File: README.md (hunk 1 above)
Comment (FayeNichols) [reply README.md:1171]: What do you think if i rename the pull request to Add FaceAPI to Machine Learning

**Review by matheusfelipeog — APPROVED** [review]
Now it's ok @FayeNichols 

Thanks for the update <3

**Pull request merged**